closed/get_openssl_source.sh

```diff
@@ -59,7 +59,8 @@ do
 	esac
 done
 
-if [ ${OPENSSL_VERSION:0:4} != "1.1." ]; then
+if [ [${OPENSSL_VERSION:0:5} != "1.0.2" ] &&
+     [${OPENSSL_VERSION:0:4} != "1.1." ] ]; then
```

Comment:
This code still causes an error to be written when executed but does not stop the code from working. 
Will fix in another PR when we have determined the real cause.

Reply:
That should be written:
> if [ "${OPENSSL_VERSION:0:5}" != "1.0.2" -a "${OPENSSL_VERSION:0:4}" != "1.1." ] ; then

Reply:
Done